datasets/mx/deputies/crawler.py

```diff
@@ -0,0 +1,66 @@
+import re
+import requests
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+def crawl_item(input_dict: dict, context: Context):
+
+    entity = context.make("Person")
+    entity.id = context.make_id(input_dict["NombreCompleto"])
+
+    entity.add("lastName", input_dict.pop("Nombre"))
+    entity.add("firstName", input_dict.pop("PrimerApellido"))
+    entity.add("name", input_dict.pop("NombreCompleto"))
```

Comment:
the apply_name helper is the preferred way to add multiple names

Reply:
watch out - I think Nombre is first name.
Also I think the best is to concatenate Primer and Segundo Appelido and put both into lastName since I don't think w can assume the order of patrymic and matronymic